plenum/test/common/test_random_string.py

```diff
@@ -0,0 +1,39 @@
+import time
+from plenum.common.util import randomString
+
+# Checks if the function randomString() is returning correct
+# length random string for various lengths
+def test_random_string1():
+    assert (len(randomString(3)) == 3), \
+        "Function randomString(3) did not return string of len 3 characters"
+    assert (len(randomString(20)) == 20), \
+        "Function randomString() did not return string of default len 20 characters"
+    assert (len(randomString(32)) == 32), \
+        "Function call randomString(32) did not return string of len 32 characters"
+    assert (len(randomString(128)) == 128), \
+        "Function randomString(128) did not return string of len 128 characters"
+    assert (len(randomString(12800)) == 12800), \
+        "Function randomString(12800) did not return string of len 12800 characters"
```

Comment:
Adding some tests with odd sizes should be done too, since the code behaves differently.